python/hidet/transforms/__init__.py

```diff
@@ -80,5 +81,6 @@ def lower(ir_module: IRModule) -> IRModule:
         rule_based_simplify_pass(),
         inline_let_stmt_pass(),
         simplify_stmt_pass(),
+        include_nccl_pass(),
```

Comment:
Later, we will use a pass to add the header information. Let's make this pass a general one and give a name like "annotate_headers" or "annotate_include_headers". Or "annotate_header_and_libs".